config/defaults/secure-open-banking/policy-evaluation-script.js

```diff
@@ -207,8 +207,10 @@ function getIdmAccessToken() {
 function findIntentType(api) {
     if (getRequestTypeAccountAndTransactions(api) != null)
         return "accountAccessIntent"
-    else if (api === "domestic-payments")
+    else if (api === "domestic-payments" || api === "domestic-payment-consents")
         return "domesticPaymentIntent"
+    else if (api === "domestic-scheduled-payments" || api === "domestic-scheduled-payment-consents")
+        return "domesticScheduledPaymentIntent"
     return null
 }
```

Comment:
Please add brackets ({}) in conditionals to make it more readable.

Reply:
I added brackets to the if clauses.